core/modules/filter/filter.module

```diff
@@ -1410,17 +1410,11 @@ function filter_process_format($element) {
  */
 function filter_get_attached($formats) {
   $attached = array();
-  $attached['library'][] = array('filter', 'filter');
 
-  foreach ($formats as $format_name => $format_info) {
-    if (isset($added[$format_name])) {
```

Comment:
The `$added` variable is not set anywhere in the function, so this if condition is useless.